lib/tmpls/require.config.js

```diff
@@ -1,9 +1,9 @@
  var jam = {data};
 
-if (typeof exports !== "undefined" && typeof module !== "undefined") {
+if (typeof exports !== 'undefined' && typeof module !== 'undefined') {
 	module.exports = jam;
-} else if (typeof require !== "undefined" && require.config) {
+} else if (typeof require !== 'undefined' && require.config) {
 	require.config(jam)
-} else if (window) {
+} else if (typeof window !== 'undefined') {
```

Comment:
http://stackoverflow.com/questions/32598971/whats-the-purpose-of-if-typeof-window-undefined